install-apt_packages.sh

```diff
@@ -12,7 +12,7 @@ apt-get update
 apt-get install -y -qq --no-install-recommends \
 	lsb-release ca-certificates wget rsync curl \
 	python-crcmod less nano vim git locales make \
-	dirmngr gnupg \
+	dirmngr parallel gnupg \
```

Comment:
We'll obviously need to cite GNU parallel, but for ease of use do we need to do anything to bypass the `parallel` citation prompt, like:
```
echo 'will cite' | parallel --citation 1> /dev/null 2> /dev/null
```
(as an alternative to passing `--will-cite` to each `parallel` invocation)